### DISCUSSION TITLE
UQL <> Infinity - fail to parse empty respnse

### DISCUSSION THREAD

**Discussion by tamirdavid** [grafana/grafana-infinity-datasource#736] [Q&A]
Hey there,
I have 2 multi-choice variables on one of my Grafana dashboards.
The values of them fetched using infinity API + UQL to parse the relevant data - until here everything is fine.

In some cases the variables are empty what cause dashboards that filter based on it to fail.
**Example of failure:**
```
parse-json
| where "Application" in (${application:singlequote})
| where "Subsystem" in (${subsystem:singlequote})
```
Removing the two where statements make it works, but I want it to work with them :)
Thanks in advance

**Comment by Wizzzzzzard** [comment]
Something closer to 

```
parse-json
| jsonata "*[Application in [${application:singlequote}]][]"
| jsonata "*[Subsystemin [${subsystem:singlequote}]][]"
```

might do the job as for us an empty value causes it to select all
